libpod/pod_api.go

```diff
@@ -541,6 +541,9 @@ func (p *Pod) Inspect() (*define.InspectPodData, error) {
 		infraConfig.CPUPeriod = p.CPUPeriod()
 		infraConfig.CPUQuota = p.CPUQuota()
 		infraConfig.CPUSetCPUs = p.ResourceLim().CPU.Cpus
+		if p.config.Pid.NSMode != "" {
+			infraConfig.Pid = p.Pid()
```

Comment:
Why is this needed?

Reply:
If I can fix `Pid()` to return the proper value I guess it won't be but right now it is never "empty" so the file path is always in inspect even when user didn't specify `--pid`